baseband/vdif/base.py

```diff
@@ -313,11 +313,11 @@ def __init__(self, fh_raw, subset=(), sample_rate=None, fill_value=0.,
         # threads there are, and what the frameset size is. For this purpose,
```

Comment:
There's now considerably more commentary than code in `__init__`, which makes reading the nested if statements difficult.  Not sure if there's a simple way to clean it up without making the comments more confusing though.

Reply:
I agree, but feel that it is best left to another PR...